salt/_modules/vault.py

```diff
@@ -363,12 +366,15 @@ def configure_auth_methods(self, client, remote_methods, local_methods, ret):
             if auth_method.extra_config:
                 log.debug(
                     'Provisioning extra configurations for auth method "%s"', auth_method.type)
-                # Get LDAP group mapping from vault
-                ldap_list_group_response = client.auth.ldap.list_groups()
-                if ldap_list_group_response:
-                    ldap_groups = ldap_list_group_response["data"]["keys"]
 
-                log.debug("LDAP groups from vault: %s", str(ldap_groups))
+                # Get LDAP group mapping from vault
+                try:
+                    ldap_groups = client.auth.ldap.list_groups()
+                    log.debug('The following groups are configured in the LDAP auth method: {groups}'.format(
+                        groups=','.join(ldap_groups['data']['keys'])
+                    ))
+                except:
```

Comment:
at a minimum, `except Exception`... if you know the actual exception, use that specifically...
```suggestion
                except Exception:
```